cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala

```diff
@@ -70,36 +72,38 @@ class FulltextIndexTest extends DocumentingTest {
           || List available analyzers            | `db.index.fulltext.listAvailableAnalyzers`                | List the available analyzers that the full-text indexes can be configured with.
           || Use full-text node index            | `db.index.fulltext.queryNodes`                            | Query the given full-text index. Returns the matching nodes and their Lucene query score, ordered by score.
           || Use full-text relationship index    | `db.index.fulltext.queryRelationships`                    | Query the given full-text index. Returns the matching relationships and their Lucene query score, ordered by score.
-          || Drop full-text index                | `db.index.fulltext.drop`                                  | Drop the specified index.
+          || Drop full-text index                | `DROP INDEX ...`                                          | Drop the specified index.
           || Eventually consistent indexes       | `db.index.fulltext.awaitEventuallyConsistentIndexRefresh` | Wait for the updates from recently committed transactions to be applied to any eventually-consistent full-text indexes.
+          || Listing all fulltext indexes        | `SHOW FULLTEXT INDEXES`                                   | Lists all fulltext indexes, see <<administration-indexes-list-indexes, the `SHOW INDEXES` command>> for details.
           ||===
           |"""
       )
     }
 
     section("Create and configure full-text indexes", "administration-indexes-fulltext-search-create-and-configure") {
-      p("""Full-text indexes are created with the `db.index.fulltext.createNodeIndex` and `db.index.fulltext.createRelationshipIndex` procedures.
-          #An index must be given a unique name when created, which is used to reference the specific index when querying or dropping it.
+      p("""Full-text indexes are created with the `CREATE FULLTEXT INDEX` command.
+          #An index can be given a unique name when created (or get a generated one), which is used to reference the specific index when querying or dropping it.
           #A full-text index applies to a list of labels or a list of relationship types, for node and relationship indexes respectively, and then a list of property names.""".stripMargin('#'))
+      p("include::create-fulltext-syntax.asciidoc[]")
```

Comment:
Looks like this include did not work, the table is not rendered

Reply:
sad will look into it

Reply:
I figured the problem at least, the target folders are not matching up with the code folders: the file I'm trying to include is in another folder than the rendering of this file ends up in. Lets hope I managed to get the correct path now :)